Cache/Requests.lean

```diff
@@ -33,8 +33,8 @@ def mkGetConfigContent (hashMap : IO.HashMap) : IO String := do
     pure $ (s!"url = {← mkFileURL fileName none}\n-o {IO.CACHEDIR / fileName}") :: acc
   return "\n".intercalate l
 
-/-- Calls `curl` to download files from the server to `CACHEDIR` (`.cache`) then unpacks them -/
-def getFiles (hashMap : IO.HashMap) (forceDownload : Bool) : IO Unit := do
+/-- Calls `curl` to download files from the server to `CACHEDIR` (`.cache`) -/
+def downloadFiles (hashMap : IO.HashMap) (forceDownload : Bool) : IO Unit := do
   let hashMap := if forceDownload then hashMap else hashMap.filter (← IO.getLocalCacheSet) false
```

Comment:
`hashMap` is redefined here, so `unpackCache hashMap` means something different if we call it at the end of this function.